src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let preparedSelectedId = selectedTabId;
+
+  if (!tabs.find(tab => tab.id === selectedTabId)) {
+    preparedSelectedId = tabs[0].id;
+  }
```

Comment:
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
better to find selectedTab and use it